libOPHD/libOPHD.vcxproj.filters

```diff
@@ -77,8 +80,8 @@
     <ClInclude Include="XmlSerializer.h">
       <Filter>Header Files</Filter>
     </ClInclude>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="..\.clang-format" />
+    <ClInclude Include="Map\MapOffset.h">
+      <Filter>Header Files\Map</Filter>
+    </ClInclude>

```

Comment:
Any idea why the reference to `.clang-format` was removed?

Or for that matter, any idea why it was there in the first place? Does having the reference there make it easy to open and edit the file? I haven't used Visual Studio in ages, so I'm not sure what the developer experience is like in that environment.

Reply:
I undid the `.clang-format` change, since it seemed to be unintentional.

I don't entirely understand why that entry is there, or if we need it, though it sounds like that's a decision to make somewhere else.